package/opt/hassbian/suites/homeassistant.sh

```diff
@@ -61,19 +61,35 @@ return 0
 }
 
 function homeassistant-upgrade-package {
-echo "Checking current version"
-pypiversion=$(curl -s https://pypi.python.org/pypi/homeassistant/json | grep '"version":' | awk -F'"' '{print $4}')
-
-sudo -u homeassistant -H /bin/bash << EOF | grep Version | awk '{print $2}'|while read -r version; do if [[ "${pypiversion}" == "${version}" ]]; then echo "You already have the latest version: $version";exit 1;fi;done
-source /srv/homeassistant/bin/activate
-pip3 show homeassistant
+if [ "$DEV" == "true"  ]; then
+  echo "This scripts downloads new scripts directly from the dev branch on github."
```

Comment:
This seems a bit out of place?

Reply:
Is it?
When the --dev flag is used, we do not need to check current version, since this change with every commit?